chronix-query-language/src/main/java/de/qaware/chronix/cql/antlr/CQLCFLexer.java

```diff
@@ -0,0 +1,139 @@
+            "T__0", "T__1", "T__2", "T__3", "T__4", "LOWERCASE_CHAR", "STRING_LITERAL"
+    };
+    /**
+     * @deprecated Use {@link #VOCABULARY} instead.
+     */
+    @Deprecated
+    public static final String[] tokenNames;
+    public static final String _serializedATN =
+            "\3\u608b\ua72a\u8133\ub9ed\u417c\u3be7\u7786\u5964\2\t\"\b\1\4\2\t\2\4" +
+                    "\3\t\3\4\4\t\4\4\5\t\5\4\6\t\6\4\7\t\7\4\b\t\b\3\2\3\2\3\3\3\3\3\4\3\4" +
+                    "\3\5\3\5\3\6\3\6\3\7\6\7\35\n\7\r\7\16\7\36\3\b\3\b\2\2\t\3\3\5\4\7\5" +
+                    "\t\6\13\7\r\b\17\t\3\2\4\3\2c|\b\2((..\60<C\\^^c|\2\"\2\3\3\2\2\2\2\5" +
+                    "\3\2\2\2\2\7\3\2\2\2\2\t\3\2\2\2\2\13\3\2\2\2\2\r\3\2\2\2\2\17\3\2\2\2" +
+                    "\3\21\3\2\2\2\5\23\3\2\2\2\7\25\3\2\2\2\t\27\3\2\2\2\13\31\3\2\2\2\r\34" +
+                    "\3\2\2\2\17 \3\2\2\2\21\22\7=\2\2\22\4\3\2\2\2\23\24\7}\2\2\24\6\3\2\2" +
+                    "\2\25\26\7\177\2\2\26\b\3\2\2\2\27\30\7<\2\2\30\n\3\2\2\2\31\32\7.\2\2" +
+                    "\32\f\3\2\2\2\33\35\t\2\2\2\34\33\3\2\2\2\35\36\3\2\2\2\36\34\3\2\2\2" +
+                    "\36\37\3\2\2\2\37\16\3\2\2\2 !\t\3\2\2!\20\3\2\2\2\4\2\36\2";
+    public static final ATN _ATN =
+            new ATNDeserializer().deserialize(_serializedATN.toCharArray());
+    protected static final DFA[] _decisionToDFA;
+    protected static final PredictionContextCache _sharedContextCache =
+            new PredictionContextCache();
+    private static final String[] _LITERAL_NAMES = {
+            null, "';'", "'{'", "'}'", "':'", "','"
+    };
+    private static final String[] _SYMBOLIC_NAMES = {
+            null, null, null, null, null, null, "LOWERCASE_CHAR", "STRING_LITERAL"
+    };
+    public static final Vocabulary VOCABULARY = new VocabularyImpl(_LITERAL_NAMES, _SYMBOLIC_NAMES);
+    public static String[] channelNames = {
+            "DEFAULT_TOKEN_CHANNEL", "HIDDEN"
+    };
+    public static String[] modeNames = {
+            "DEFAULT_MODE"
+    };
+
+    static {
+        RuntimeMetaData.checkVersion("4.7.1", RuntimeMetaData.VERSION);
+    }
+
+    static {
+        tokenNames = new String[_SYMBOLIC_NAMES.length];
+        for (int i = 0; i < tokenNames.length; i++) {
+            tokenNames[i] = VOCABULARY.getLiteralName(i);
+            if (tokenNames[i] == null) {
+                tokenNames[i] = VOCABULARY.getSymbolicName(i);
+            }
+
+            if (tokenNames[i] == null) {
+                tokenNames[i] = "<INVALID>";
+            }
+        }
+    }
+
+    static {
+        _decisionToDFA = new DFA[_ATN.getNumberOfDecisions()];
+        for (int i = 0; i < _ATN.getNumberOfDecisions(); i++) {
+            _decisionToDFA[i] = new DFA(_ATN.getDecisionState(i), i);
+        }
+    }
+
+    public CQLCFLexer(CharStream input) {
+        super(input);
+        _interp = new LexerATNSimulator(this, _ATN, _decisionToDFA, _sharedContextCache);
+    }
+
+    @Override
```

Comment:
\[Checkstyle\] ERROR: Must include both @java\.lang\.Deprecated annotation and @deprecated Javadoc tag with description\.